### PR TITLE
Add Swift Package Manager installation instructions to Readme

### PR DESCRIPTION
**Pull request by jameslittle230**
Lots of packages in other communities have installation instructions (even if it's just a Terminal one-liner) in the Readme to help people who might not have much familiarity with the ecosystem. When I was installing this package, I realized I wanted similar instructions in *this* package's Readme, so this PR adds them.

Feel free to edit or reject, I'm not all that experienced with open source contributing

### CODE REVIEW

**Comment by BalestraPatrick** [comment]
Hey James! Great idea, I completely forgot to add installation instructions. Can you also add CocoaPods as I did in one of my other projects? https://github.com/BalestraPatrick/WhatsNew#installation

You can rename the section to `Installation` and have subsections for SPM and CocoaPods :)

I'll be happy to review the PR again, just mention me and I'll get back to you much faster compared to this time!

**Comment by jameslittle230** [comment]
@BalestraPatrick Let me know if you're not happy with the wording!

File: README.md [hunk 1]
```diff
@@ -110,6 +110,32 @@ public subscript(range: PartialRangeFrom<String>) -> PartialRangeFrom<String.Ind
 public subscript(range: PartialRangeThrough<String>) -> PartialRangeThrough<String.Index>?
 ```
 
+## Installation
+
+### Cocoapods
+
+To install via [Cocoapods](http://cocoapods.org/), add the following line to your Podfile:
+
+```
```
Comment (BalestraPatrick) [inline README.md]: Make this ```ruby to have syntax highlighting

File: README.md [hunk 2]
```diff
@@ -110,6 +110,32 @@ public subscript(range: PartialRangeFrom<String>) -> PartialRangeFrom<String.Ind
...
+pod Stryng
+```
+
+### Swift Package Manager
+
+To install via the [Swift Package Manager](https://swift.org/package-manager/), add the following line to the `dependencies` array in your `Package.swift` file:
+
+```
```
Comment (BalestraPatrick) [inline README.md]: Make this ```swift to have syntax highlighting

File: README.md [hunk 3]
```diff
@@ -110,6 +110,32 @@ public subscript(range: PartialRangeFrom<String>) -> PartialRangeFrom<String.Ind
...
+.package(url: "https://github.com/BalestraPatrick/Stryng.git", from: "0.4.1")
+```
+
+Then, still in your `Package.swift`, add `"Stryng"` to your *target's* `dependencies` array.
+
+Finally, in your terminal, run the following command to update your dependencies:
+
+```
+$ swift package update
```
Comment (BalestraPatrick) [inline README.md:136]: Make this ```bash to have syntax highlighting

**Comment by BalestraPatrick** [comment]
@jameslittle230 Just some minor fixes and then this is ready to be merged 👍

**Comment by jameslittle230** [comment]
@BalestraPatrick Syntax has been highlighted!

**Comment by BalestraPatrick** [comment]
Looks good to me, thanks a lot for your contribution @jameslittle230! 🎉

**Pull request merged**